### PR TITLE
Update docker-compose from make_release.sh

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

#2398

# Overview

- e02d604bdced29f0ff2cd38f55f053fd2baab300
  Update docker-compose from make_release.sh

# Comments

### CODE REVIEW

File: scripts/make_release.sh [hunk 1]
```diff
@@ -38,11 +38,15 @@ WIKI_COMMIT=$(git ls-remote https://github.com/$REPO.wiki.git HEAD | cut -f1)
 echo ""
 echo "Replacing $OLD_CABAL_VERSION with $CABAL_VERSION"
 sed -i "s/$OLD_CABAL_VERSION/$CABAL_VERSION/" \
-    $(git ls-files '*.nix:!:nix/migration-tests.nix'; git ls-files '*.cabal'; git ls-files '*swagger.yaml')
+    $(git ls-files '*.nix:!:nix/migration-tests.nix'; git ls-files '*.cabal'; git ls-files '*swagger.yaml'; git ls-files 'docker-compose.yml')
```
Comment (rvl) [inline scripts/make_release.sh]:
```suggestion
    $(git ls-files '*.nix:!:nix/migration-tests.nix'; git ls-files '*.cabal'; git ls-files '*swagger.yaml') docker-compose.yml
```

File: scripts/make_release.sh [hunk 2]
```diff
@@ -38,11 +38,15 @@ WIKI_COMMIT=$(git ls-remote https://github.com/$REPO.wiki.git HEAD | cut -f1)
...
 echo "Looking for remaining references to old version:"
 git grep $OLD_CABAL_VERSION
 echo ""
 
+echo "Updating docker-compose.yml with $CARDANO_NODE_TAG cardano-node tag"
+sed -i "s/$OLD_CARDANO_NODE_TAG/$CARDANO_NODE_TAG/" docker-compose.yml
```
Comment (rvl) [inline scripts/make_release.sh]: This way you don't need the old tag - 

```suggestion
sed -i "s|inputoutput/cardano-node:.*|inputoutput/cardano-node:$CARDANO_NODE_TAG|" docker-compose.yml
```

**Review by rvl — APPROVED** [review]
Good idea

**Comment by piotr-iohk** [comment]
Thanks @rvl. Fixed.

**Review by KtorZ — APPROVED** [review]

**Pull request merged**